must-gather/node-gather/node-gather-ds.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-gather-daemonset
+  namespace: node-gather
+  labels:
+spec:
+  selector:
+    matchLabels:
+      name: node-gather-daemonset
+  template:
+    metadata:
+      labels:
+        name: node-gather-daemonset
+    spec:
+      serviceaccount: node-gather
+      serviceAccountName: node-gather
+      terminationGracePeriodSeconds: 0
+      hostNetwork: true
+      containers:
+      - name: node-probe
```

Comment:
I don't think we care about any of this do we?

Reply:
The way that the feature is implemented: oc creates a pod on an image as specified by the --image cmd line argument to oc admin must-gather. Now the container is expected to have the node-probe container,  HostNetworking must be true - otherwise our container can't obtain any informaiton. terminatioGracePeriodSeconds - also mandated by how oc is using this image.

Reply:
(comment to earlier review)
I think this is a misunderstanding:
The 2 manifests in `node-gather` are used by kubevirt's must gather to deploy a daemonset while collecting logs etc. A script is using this daemonset for executing commands on the nodes, see https://github.com/kubevirt/must-gather/blob/master/collection-scripts/gather_nodes#L52
As long as we don't need to collect info from nodes which we can't get with the `oc` command, we don't need these manifests at all.

Reply:
removed the daemon set.